src/sentry/static/sentry/app/views/groupDetails/userWidget.jsx

```diff
@@ -8,28 +27,25 @@ var UserWidget = React.createClass({
 
   render() {
     var user = this.props.data;
+    var children = [];
+
+    // Handle our native attributes special
+    user.id && children.push(renderLine('id', user.id, 'ID'));
+    user.email && children.push(renderLine('email', user.email));
+    user.username && children.push(renderLine('username', user.username));
+    user.ipAddress && children.push(renderLine('ipAddress', user.ipAddress, 'IP'));
+
+    // We also attach user supplied data as 'user.data'
+    _.each(user.data, function(value, key) {
+      children.push(renderLine(key, value));
+    });
```

Comment:
You can leave it. The 'ID' / 'IP' thing makes it less pretty. I also left out `user.data`.